### ISSUE TITLE
Configuration for rule "no-unused-vars" is invalid

### ISSUE THREAD

**Issue by MiguelArmendariz** [wesbos/eslint-config-wesbos#51]
I've been using this config for a while and a couple of weeks ago the eslint stopped working and kinda the intellisense paths on import, so I thought it was an issue from vs code but when I try to run the eslint command I'm having this issue.

```
Error: .eslintrc » eslint-config-wesbos:
        Configuration for rule "no-unused-vars" is invalid:
        Value {"ignoreSiblings":true,"argsIgnorePattern":"res|next|^err"} should be equal to one of the allowed values.
        Value {"ignoreSiblings":true,"argsIgnorePattern":"res|next|^err"} should NOT have additional properties.
        Value {"ignoreSiblings":true,"argsIgnorePattern":"res|next|^err"} should match exactly one schema in oneOf.
```

Supposedly at the next files from node-modules/eslint/*: 

```
config-validator.js:132:19
config-validator.js:187:9
config-validator.js:184:30
config-validator.js:312:9
cascading-config-array-factory.js:464:13
cascading-config-array-factory.js:275:21
file-enumerator.js:431:49
file-enumerator.js:287:49
```

**Comment by wesbos** [comment]
Hm that is weird. What version of eslint are you using?

**Comment by tm1000** [comment]
Issue is fixed here in this PR: https://github.com/wesbos/eslint-config-wesbos/pull/50

**Comment by tm1000** [comment]
@wesbos Thank you for merging! this single issue was preventing us from upgrading ESLint but I never looked into the why until now. Can you issue a release as well?

**Comment by wesbos** [comment]
fixed in 0.0.20

**Comment by MiguelArmendariz** [comment]
> Hm that is weird. What version of eslint are you using?

Sorry I didn't answer I was away this weekend. Thanks for the help tho.
